tests/unit/modules/test_junos.py

```diff
@@ -0,0 +1,1679 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import
+from __future__ import print_function
+
+__author__ = "Rajvi Dhimar"
+
+import unittest
+from nose.plugins.attrib import attr
```

Comment:
We don't use nose and we're actually working towards using pytest.
`attr` seems just like a test selection decorator and, if that's the only benefit, could it be removed?

You can select this test case on the develop branch by passing the module filesystem path to runtests.py